jobs/update_orphaning_dashboard_etl.py

```diff
@@ -186,7 +149,7 @@ def longitudinal_shim_aggregate(date_from, date_to, destination_project, destina
             `moz-fx-data-shared-prod.telemetry.main`
         WHERE
             sample_id = 42
-            AND DATE(submission_timestamp)>='2019-09-01' )
+            AND environment.build.version IS NOT NULL )
```

Comment:
Also, will this run without a partition filter on `telemetry.main`?

Reply:
Yes, this line is a CTE, there is a partition filter couple o lines below.